src/name_bazaar/server/syncer.cljs

```diff
@@ -26,11 +26,9 @@
 
 (defn node-owner? [offering-address {:keys [:offering/node :offering/top-level-name? :offering/label]}]
   (let [ens-owner (ens/owner {:ens.record/node node})]
-    (and (= ens-owner offering-address)
-         (if top-level-name?
-           (= (registrar/registration-owner {:ens.record/label label})
-              offering-address)
-           true))))
+    (if top-level-name?
```

Comment:
The registration transfer is a must, since registration owner can reclaim ENS ownership at any time. ENS ownership tranfer is a courtesy. Ceasing to require ENS ownership at inopportune times enables us to transfer ownership into offering in one transaction only, which is the desired result.